downstream/modules/eda/proc-eda-set-up-new-decision-environment.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc-eda-set-up-new-decision-environment"]
+
+= Setting up a new decision environment
+
+Decision environments are a container image to run Ansible rulebooks.
+They create a common language for communicating automation dependencies, and provide a standard way to build and distribute the automation environment.
+The default decision environment is found in the link:https://quay.io/repository/ansible/ansible-rulebook[Ansible-Rulebook]. 
+
+To create your own decision environment refer to link:https://access.redhat.com/solutions/7018085[How do I Build a Custom Decision Environment for Event Driven Ansible within Ansible Automation Platform?].
+
+The following steps describe how to import a decision environment into your {EDAcontroller} Dashboard.
+
+== Prerequisites
+
+* You are logged in to the {EDAcontroller} Dashboard as a Content Consumer.
+* You have set up a credential, if necessary. 
+For more information, refer to the link:https://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html[Credentials] section
+of the {ControllerNameStart} documentation.
+* You have pushed a decision environment image to an image repository or you chose to use the image `de-supported` provided at link:http://registry.redhat.io/[registry.redhat.io].
+
+== Procedure
+
+. Navigate to the {EDAcontroller} Dashboard.
+. From the navigation panel, select menu:Decision Environments[Create decision environment].
+. Insert the following: 
++
+Name::
+Description:: This field is optional.
+Image:: This is the full image location, including the container registry, image name, and version tag.
+Credential:: This is the token needed to utilize the decision environment image. 
```

Comment:
Should this indicate that this field is optional? To be more precise the credential for DEs is a pair of user/token

Reply:
This is how it currently appears in the UI as per Tina's mockups. Will confirm with her if this should be optional.

Reply:
If it should be optional, I can update the mocks.

Reply:
@Alex-Izquierdo Should the Credential field in Projects also be optional as well then?

Reply:
Correct. Credentials should be optional for projects and decision environments. It is actually in the UI and mocks should be updated.